docker/mysqldb.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.1'
```

Comment:
Hey Al, taking a second look at this PR, I realised that you only added the `mysql` container to this compose yml. If you take a look at the `docker-compose.yml` in the project repo, you can see that I start both the `urchin` app and the `mariadb` app.

Do you want to move some of that logic to this? Also, let's start a `mariadb` image as that's what we use for development.

Reply:
As mentioned, the `mariadb.yml` should be used only for local development. I moved the docker-compose file for full set-up into the `docker` directory and added all the files related to starting the application in a docker container within the same directory.

When developing you have now the option to either start the application in a full docker setup and work locally, or you can just spin up a development database and run the urchin from within VSCode with the given launch.json.

Reply:
I like it, I agree with the changes. I didn't realise that it would be easier to spin up the container just with the Database.